nixos/modules/services/misc/klipper.nix

```diff
@@ -35,6 +35,38 @@ in
         description = lib.mdDoc "Path of the API socket to create.";
       };
 
+      isConfigMutable = mkOption {
+        type = types.bool;
+        default = false;
+        example = true;
+        description = lib.mdDoc ''
+          Whether to copy config to mutable folder instead of using one directly from the store.
+          This will only copy config from NixOS configuration if file at `services.klipper.mutableConfigPath` doesn't exist.
```

Comment:
```suggestion
          Whether to copy the config to a mutable directory instead of using the one directly from the nix store.
          This will only copy the config if the file at `services.klipper.mutableConfigPath` doesn't exist.
```